docs/userstories.rst

```diff
@@ -0,0 +1,37 @@
+.. _userstories:
+
+User Stories
+============
+
+TODO: introductory text
+
+.. contents::
+
+.. _usera:
+User A
+------
+
+Writes application code on top of well-established open-source software libraries
+to tackle domain-specific problems. Moreover, User A writes scripts for pre-
+and post-processing of input/output data. The written code is domain- and
+project-specific and does not have substantial potential for reusability.
+
+.. _userb:
+User B
+------
+
+Helps in the development and maintenance of an open-source software framework.
+User B's research activities involve the development of new algorithms, which are
+implemented into the framework in the scope of research projects, and which are
```

Comment:
also, this is probably way too specific. For the workflow aspect, it doesn't really matter if the algorithms are published in/implemented into the framework.